protocol/x/clob/keeper/final_settlement.go

```diff
@@ -0,0 +1,71 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	indexerevents "github.com/dydxprotocol/v4-chain/protocol/indexer/events"
+	"github.com/dydxprotocol/v4-chain/protocol/indexer/indexer_manager"
+	indexershared "github.com/dydxprotocol/v4-chain/protocol/indexer/shared"
+	"github.com/dydxprotocol/v4-chain/protocol/x/clob/types"
+)
+
+// MustEnterFinalSettlement holds logic executed when a market transitions to FINAL_SETTLEMENT status.
+// This function will forcefully cancel all stateful open orders for the clob pair.
+func (k Keeper) mustEnterFinalSettlement(ctx sdk.Context, clobPairId types.ClobPairId) {
```

Comment:
Suggestion, up to you:
```suggestion
func (k Keeper) mustTransitionToFinalSettlement(ctx sdk.Context, clobPairId types.ClobPairId) {
```